posthog/clickhouse/migrations/0185_events_historical_migration_column.py

```diff
@@ -0,0 +1,33 @@
+from posthog.clickhouse.client.connection import NodeRole
+from posthog.clickhouse.client.migration_tools import run_sql_with_exceptions
+
+ADD_HISTORICAL_MIGRATION_COLUMN_EVENTS = """
+ALTER TABLE {table}
+ADD COLUMN IF NOT EXISTS historical_migration Bool
+"""
+
+ADD_HISTORICAL_MIGRATION_COLUMN_INDEX = """
+ALTER TABLE {table}
+ADD INDEX IF NOT EXISTS historical_migration_minmax (historical_migration) TYPE minmax GRANULARITY 1
+"""
+
+operations = [
+    run_sql_with_exceptions(
+        ADD_HISTORICAL_MIGRATION_COLUMN_EVENTS.format(table="sharded_events"),
+        node_roles=[NodeRole.DATA],
+        sharded=True,
+        is_alter_on_replicated_table=True,
+    ),
+    run_sql_with_exceptions(
+        ADD_HISTORICAL_MIGRATION_COLUMN_INDEX.format(table="sharded_events"),
+        node_roles=[NodeRole.DATA],
+        sharded=True,
+        is_alter_on_replicated_table=True,
+    ),
+    run_sql_with_exceptions(
+        ADD_HISTORICAL_MIGRATION_COLUMN_EVENTS.format(table="events"),
+        node_roles=[NodeRole.COORDINATOR],
```

Comment:
`events` table in US will miss the `historical_migration` shouldn't it be data+coord?

```suggestion
        node_roles=[NodeRole.COORDINATOR, NodeRole.DATA],
```